solc/CommandLineInterface.cpp

```diff
@@ -204,9 +204,9 @@ static bool needsHumanTargetedStdout(po::variables_map const& _args)
 {
 	if (_args.count(g_argGas))
 		return true;
-	if (_args.count(g_argOutputDir))
```

Comment:
With this, the semantics of the function changes and it should now rather be called `canHaveHumanTargetedStdout`

Reply:
Done.